release-guide.md

```diff
@@ -0,0 +1,104 @@
+# SIARD Suite Release Testing Guide
+
+## Motivation
+
+The development of SIARD Suite is financed by the Swiss Federal Archives (SFA), which is one of the main stakeholders of the project. The SFA is also one of the main users of the application. There are many restrictions on the workstations that can be used by SFA staff:
+
+- Pre-installed Java 8 (+JavaFX) with no possibility to switch to another JRE.
+- Inability to download and run any form of executable binary or script (e.g. `SIARD-suite.bat`).
+
+The only way SFA staff can run the application is with the following cli command
+
+```bash
+java -jar lib/siard-suite.jar
+```
+
+Due to other constraints, it has happened on a regular basis that the application runs fine when using the native installers or start scripts (with bundled jre), but not with the above command, without anyone noticing until an SFA staff member tries to run the application without success.
+
+That's why it is recommended to perform the following steps to make sure that the application really works for SFA staff and users of different distributions (native installers).
+
+
+## Test script
+
+First, run a build and create native installers and packaged distributions:
+
+```bash
+./gradlew clean jpackage
+```
+
+Start some databases to do some smoke tests:
+
+```bash
+docker-compose up -d --build
+```
+
+
+### Smoke test for SFA environment
+
+Switch to the unpacked distribution of your application:
+
+```bash
+cd build/install/SIARD-Suite/
+```
+
+Use Java 8 with JavaFX via asdf:
+
+```
+asdf local java zulu-jre-javafx-8.76.0.17
+```
+
+Note: If you don't use asdf, but e.g. sdk man or some other way to switch between Java versions: it is important to use a java8 jre with javafx in this step!
```

Comment:
Note: If you don't use asdf, but sdkman or any  other way of switching between Java versions, it is important to use a Java 8 JRE with JavaFX in this step